laserstudio/instruments/instruments.py

```diff
@@ -96,6 +97,17 @@ def __init__(self, config: dict):
                     continue
                 self.probes.append(ProbeInstrument(config=probe_config))
 
+        # Hayashi Light Remote
+        self.hayashi_light: Optional[HayashiLRInstrument] = None
+        hayashi_config = config.get("hayashi", None)
+        if hayashi_config is not None and hayashi_config.get("enable", False):
```

Comment:
In another PR (for the shutter) I see according to the schema instruments are enabled by default, so maybe it should be  `if hayashi_config is not None and hayashi_config.get("enable", True):` instead?